config/default/pre-commit-config.yaml.j2

```
@@ -71,6 +71,17 @@ repos:
     rev: "6.0.0"
     hooks:
     -   id: i18ndude
+"%(i18ndude_extra_lines)s"
```

```suggestion
%(i18ndude_extra_lines)s
```

No quotes, otherwise it will break the configuration